app/routes/listen.js

```diff
@@ -32,6 +34,7 @@ export default Route.extend({
       let controller = this.controllerFor('application');
       Ember.run.scheduleOnce('afterRender', this, function() {
         controller.send('updateNav');
+        get(this, 'dataLayer').sendPageView();
```

Comment:
shouldn't need to use a getter in the latest embers
```suggestion
        this.dataLayer.sendPageView();
```

Reply:
also yo might be better served with a single `routeDidChange` event handler set up in your application route, so that you aren't adding in a pageview event for every route by hand, similar to this

https://github.com/nypublicradio/gothamist-web-client/blob/6263c6212a293d931499b8cc370776015c32ab1a/app/routes/application.js#L36-L45

Reply:
Great, I was looking for something like this.